### PR TITLE
Update docs with new details for 3g

### PR DESCRIPTION
**Pull request by ImmaZoni**

### CODE REVIEW

File: versioned_docs/version-latest/operators_and_staking/staking.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Once the withdrawal is submitted, it's finalized after the **domain epoch** is c
 
 1. Proceed to PolkadotJS **[Network Explorer](https://polkadot.js.org/apps/#/explorer)**.
 2. Go to `Developer` -> `Chain state`.
-3. Select `domains` under `selected state query` and `nominators(u64, AccoundID32)`.
+3. Select `domains` under `selected state query` and `nominators(u64, AccoundId32)`.
```
Comment (jim-counter) [inline versioned_docs/version-latest/operators_and_staking/staking.md:117]:
```suggestion
3. Select `domains` under `selected state query` and `nominators(u64, AccountId32)`.
```

File: versioned_docs/version-latest/protocol/substrate-cli.mdx [hunk 2]
```diff
@@ -371,6 +371,8 @@ We will be downloading two files for your respective operating system.
   2022-02-03 10:52:29 ⚙️  Syncing, target=#215883 (2 peers), best: #55 (0xafc7…bccf), finalized #0 (0x6ada…0d38), ⬇ 850.1kiB/s ⬆ 1.5kiB/s
   ```
   5. We will then open another terminal, change to the downloads directory, then start the farmer node with the following command:
+  :::note Farming while Plotting changes
```
Comment (jim-counter) [inline versioned_docs/version-latest/protocol/substrate-cli.mdx:374]: I can't see this in the preview?

File: versioned_docs/version-latest/protocol/substrate-cli.mdx [hunk 3]
```diff
@@ -902,13 +904,23 @@ There are extra commands and parameters you can use on farmer or node, use the `
 ## IV. Advanced
 
 ---
+### Timekeepers
+
+Gemini-3g Introduces our Proof-of-Time, a new optional role that has been added to the node. The Timekeepers help run PoT on the network to ensure the security of the network. Timekeeping requires a high-performance core cpu but can be ran by any node runner. You can enable time keeping with the following commands.
```
Comment (jim-counter) [inline versioned_docs/version-latest/protocol/substrate-cli.mdx:909]:
```suggestion
Gemini-3g introduces Proof-of-Time and a new, optional role has been added to the node. Timekeepers help run PoT to ensure the security of the network. Timekeeping requires a high-performance core CPU but can be undertaken by any node runner. You can enable timekeeping with the following commands.  
```

File: versioned_docs/version-latest/protocol/substrate-cli.mdx [hunk 4]
```diff
@@ -902,13 +904,23 @@ There are extra commands and parameters you can use on farmer or node, use the `
 ## IV. Advanced
 
 ---
+### Timekeepers
+
+Gemini-3g Introduces our Proof-of-Time, a new optional role that has been added to the node. The Timekeepers help run PoT on the network to ensure the security of the network. Timekeeping requires a high-performance core cpu but can be ran by any node runner. You can enable time keeping with the following commands. 
+- `--timekeeper`: To Enable Timekeeping on the node
```
Comment (jim-counter) [inline versioned_docs/version-latest/protocol/substrate-cli.mdx:910]:
```suggestion
- `--timekeeper`: To enable timekeeping on the node
```

File: docs/protocol/substrate-cli.mdx [hunk 5]
```diff
@@ -371,6 +371,8 @@ We will be downloading two files for your respective operating system.
   2022-02-03 10:52:29 ⚙️  Syncing, target=#215883 (2 peers), best: #55 (0xafc7…bccf), finalized #0 (0x6ada…0d38), ⬇ 850.1kiB/s ⬆ 1.5kiB/s
   ```
   5. We will then open another terminal, change to the downloads directory, then start the farmer node with the following command:
+  :::note Farming while Plotting changes
```
Comment (jim-counter) [inline docs/protocol/substrate-cli.mdx:374]: I'm not sure what's going on here? I can't see this change in the preview?

File: docs/protocol/substrate-cli.mdx [hunk 6]
```diff
@@ -902,13 +904,23 @@ There are extra commands and parameters you can use on farmer or node, use the `
 ## IV. Advanced
 
 ---
+### Timekeepers
+
+Gemini-3g Introduces our Proof-of-Time, a new optional role that has been added to the node. The Timekeepers help run PoT on the network to ensure the security of the network. Timekeeping requires a high-performance core cpu but can be ran by any node runner. You can enable time keeping with the following commands. 
+- `--timekeeper`: To Enable Timekeeping on the node
```
Comment (jim-counter) [inline docs/protocol/substrate-cli.mdx:910]:
```suggestion
- `--timekeeper`: To enable Timekeeping on the node
```

File: versioned_docs/version-latest/protocol/substrate-cli.mdx [hunk 7]
```diff
@@ -902,13 +904,23 @@ There are extra commands and parameters you can use on farmer or node, use the `
...
+- `--timekeeper-cpu-cores`: To specify which cores the Timekeeper will run on.
```
Comment (jim-counter) [inline versioned_docs/version-latest/protocol/substrate-cli.mdx:911]:
```suggestion
- `--timekeeper-cpu-cores`: To specify which cores the timekeeper will run on. 
```

File: versioned_docs/version-latest/protocol/substrate-cli.mdx [hunk 8]
```diff
@@ -902,13 +904,23 @@ There are extra commands and parameters you can use on farmer or node, use the `
...
+
 
 ### Node & Farmer Commands Guide
 
 Both the node and the farmer have a variety of flags and parameters. To see a full list, append the `--help` flag to either the node or the farmer command.
 
 ### Common Command Examples
 
+:::note Farming Changes
+Please note that as of `Gemini-3g` Farming no longer occurs while plotting. This is to ensure the plotting process occurs in the most efficient manner. You can change this by adding the `--farm-during-initial-plotting` flag to the farmer.
```
Comment (jim-counter) [inline versioned_docs/version-latest/protocol/substrate-cli.mdx:921]:
```suggestion
Please note that as of `Gemini-3g` farming no longer occurs while plotting. This is to ensure the plotting process occurs in the most efficient manner. You can change this by adding the `--farm-during-initial-plotting` flag to the farmer. 
```

**Review by jim-counter — CHANGES_REQUESTED** [review]

File: docs/protocol/substrate-cli.mdx (hunk 5 above)
Comment (ImmaZoni) [reply docs/protocol/substrate-cli.mdx]: Fixed with commit 00d53e2

File: versioned_docs/version-latest/protocol/substrate-cli.mdx (hunk 2 above)
Comment (ImmaZoni) [reply versioned_docs/version-latest/protocol/substrate-cli.mdx]: fixed with commit 00d53e2

**Review by jim-counter — APPROVED** [review]
LGTM!

**Pull request merged**